kuma/wiki/views/list.py

```diff
@@ -108,6 +114,7 @@ def with_localization_tag(request, tag=None):
     return render(request, 'wiki/list/with_localization_tags.html', context)
 
 
+@block_user_agents
```

Comment:
Should we add https://developer.mozilla.org/en-US/docs/with-errors to robots.txt?